packages/field/test-suites/FormatMixin.suite.js

```diff
@@ -0,0 +1,394 @@
+import { expect, fixture, html, aTimeout, defineCE, unsafeStatic } from '@open-wc/testing';
+import sinon from 'sinon';
+
+import { LitElement } from '@lion/core';
+import { Unparseable } from '@lion/validate';
+import { FormatMixin } from '../src/FormatMixin.js';
+
+function mimicUserInput(formControl, newViewValue) {
+  formControl.value = newViewValue; // eslint-disable-line no-param-reassign
+  formControl.inputElement.dispatchEvent(new CustomEvent('input', { bubbles: true }));
+}
+
+export function runFormatMixinSuite(customConfig) {
+  // TODO: Maybe remove suffix
```

Comment:
Suffix is meant to make your test output a bit more readable. Wrapping a suite in a describe at least makes the tests 'traceable', so omitting the suffix would be fine by me if it makes our suites simpler.